src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -950,4 +952,21 @@ public void waitForVASTRequestWithAdUnit(NetworkTrafficInterceptor networkTraffi
     final String pattern = ".*output=xml_vast.*iu=" + encodedAdUnit + ".*";
     wait.forSuccessfulResponseByUrlPattern(networkTrafficInterceptor, pattern);
   }
+
+  public String getValueFromTracking(NetworkTrafficInterceptor networkTrafficInterceptor, String slotName, String paramName) {
+    final String fvTrackingUrl = getTrackingUrl(networkTrafficInterceptor, slotName);
+    Matcher matcher = Pattern.compile(".*" + paramName + "=([^&]+)&.*").matcher(fvTrackingUrl);
+    return matcher.matches() ? matcher.group(1) : "";
+  }
+
+  public String getFVLineItem() {
+    final By FV_SLOT_SELECTOR = By.cssSelector("div.featured-video__player-container");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00117)